examples/cymbal-bank/6-appsource/cymbal-bank/frontend_cloudbuild.yaml

```diff
@@ -11,24 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 steps:
   - name: gcr.io/cloud-builders/gsutil
-    id: download-skaffold-cache
```

Comment:
Why remove the ids?

Reply:
Actually this file is now, removed... it was not being used apparently